pkgs/development/libraries/zlib/default.nix

```diff
@@ -17,7 +16,7 @@ stdenv.mkDerivation rec {
     sha256 = "c3e5e9fdd5004dcb542feda5ee4f0ff0744628baf8ed2dd5d66f8ca1197cb1a1";
   };
 
-  postPatch = stdenv.lib.optionalString stdenv.isDarwin ''
+  postPatch = stdenv.lib.optionalString hostPlatform.isDarwin ''
     substituteInPlace configure \
       --replace '/usr/bin/libtool' 'ar' \
```

Comment:
@copumpkin you're looking at the person that just ran `s/stdenv\.is/hostPlatform.is/g' :D.

In seriousness, looks like `libtool` is being replaced with `ar`, which is kind of sketchy? I'd guess

 - libtool should continue to be used where possible

 - `ar` should be replaced with `$AR`